examples/persistent-storage/README.md

```diff
@@ -0,0 +1,153 @@
+# CHIP EFR32 Lighting Example
+
+An example testing and demonstrating the key value storage API.
+
+<hr>
+
+-   [CHIP EFR32 Lighting Example](#chip-efr32-lighting-example)
+    -   [Introduction](#introduction)
+    -   [EFR32](#efr32)
+        -   [Building](#building)
+        -   [Flashing the Application](#flashing-the-application)
+        -   [Viewing Logging Output](#viewing-logging-output)
+
+<hr>
+
+<a name="intro"></a>
+
+## Introduction
+
+This example serves to both test the key value storage implementation and API as
+it is brought-up on different platforms, as well as provide an example for how
+to use the API.
+
+In the future this example can be moved into a unit test when available on all
+platforms.
```

Comment:
do we have an issue covering other platforms?  can you file one (or one for each platform), as well as an issue to move this test?

Reply:
From what I understand we don't yet have a test runner on EFR3, created an issue to track moving it when available: https://github.com/project-chip/connectedhomeip/issues/4406